### DISCUSSION TITLE
No way to turn on light mode :D ?

### DISCUSSION THREAD

**Discussion by asgeirtj** [zenangst/KeyboardCowboy#492] [General]
No way to turn on light mode :D ?

**Comment by zenangst** [comment]
Hey @asgeirtj, not at the moment. I took the lazy route when designing the UI... so we are stuck in the darkness. It is on my todo-list to add proper support for light mode. Don't know when it will arrive. Currently, I'm investing adding macro support for Keyboard Cowboy, that is at the top of the pile of a bunch of other things.

But the UI is driven partially by the framework `Bonzai`. This should make it easier to support it in the future. I just need to do the work 😶

**Comment by asgeirtj** [comment]
NP at all friend. I was wondering can you use the UI element command to
press the go to bottom arrow which folloews the prompt in chatgpt for you?

Kveðja / Regards, Ásgeir

**Reply by zenangst** [reply]
I'm unsure how well this works on web content. I've mainly been using (and implementing it) for native apps. I guess you could just try and see what happens?

The UI commands depend on a somewhat unique identifier for it to be able to find the element in the "DOM"-tree.

**Comment by asgeirtj** [comment]
what are these macros you speak of? i don't find it?

Kveðja / Regards, Ásgeir

**Reply by zenangst** [reply]
You can find more information about it in the wiki - https://github.com/zenangst/KeyboardCowboy/wiki/Commands#macros

**Comment by zevrix** [comment]
I wonder if you'd be able to just provide a switch for the light mode here in `AppDelegate.swift`:

```
final class AppDelegate: NSObject, NSApplicationDelegate {
  func applicationDidFinishLaunching(_ aNotification: Notification) {
    NSApp.appearance = NSAppearance(named: .darkAqua)
  }
}
```

It doesn't seem to be an Xcode project right? Otherwise I could try it myself.

**Reply by zenangst** [reply]
@zevrix Yeah, the pinning it to dark mode is on purpose because the custom UI still doesn’t look right in light mode. As it was designed primarily for dark mode.

The project uses tuist to generate a Xcode project. Check the README for more details.

**Reply by zevrix** [reply]
ok I see, thanks. yeah i know nothing about tuist right now, may try later. I'm one of those who get totally lost in dark UI. interesting product i do want to explore it when i get a chance.

**Reply by zenangst** [reply]
Small updated, I've been working on light mode... it will make its way into production soon... ish 😎

**Comment by asgeirtj** [comment]
love it :D

Kveðja / Regards, Ásgeir

**Comment by asgeirtj** [comment]
Your app allows you to set up chorded keyboard shortcuts right? like in
vscode cmd+k+s. i can't work out how to do it here is my shortcut:[image:
image.png]

am i close?

Kveðja / Regards, Ásgeir

**Reply by zenangst** [reply]
Seems like you forgot to attach the image.
Mind posting it again?

**Reply by asgeirtj** [reply]
![image](https://github.com/zenangst/KeyboardCowboy/assets/27446620/062b18ca-361f-402c-9c90-1360a8208344)
here it is. Btw I found out how to do it 
![image](https://github.com/zenangst/KeyboardCowboy/assets/27446620/79c02169-5a0d-4cf2-9198-b0badaa61eb3)

**Reply by zenangst** [reply]
Nice, looking good.
Bundled shortcuts opens up a whole new world of possibilities 🤩
